### PR TITLE
Migrate node jobs

### PR DESCRIPTION
**Pull request by krzyzacy**
after https://github.com/kubernetes/test-infra/pull/4324

(but now it comes the code freeze, maybe I need to leave the main kubelet job here)

/assign @yguo0905

### CODE REVIEW

**Comment by yguo0905** [comment]
One question, which will not block this PR.

Is it possible to use the configs in test-infra instead of the main repo for these jobs? E.g, `--image-config-file=jobs/e2e_node/benchmark-config.yaml`.

**Comment by krzyzacy** [comment]
yeah good point, I'll do that

**Comment by krzyzacy** [comment]
@yguo0905 I copied the image configs over to `jobs/node/*.yaml` for now. This PR only touches master jobs - for branch jobs I'll make a branched image, and we'll probably also want to branch the image config (if they are different) for different release branches as well here.

**Comment by yguo0905** [comment]
/lgtm
Thanks!

**Comment by krzyzacy** [comment]
(not touching the node-1.8 job yet, I'll go ahead and make sure those master jobs will pass to start with)

File: jobs/config.json [hunk 1]
```diff
@@ -9229,30 +9240,53 @@
...
     ]
   },
   "ci-kubernetes-node-kubelet-conformance": {
     "args": [
-      "--properties=test/e2e_node/jenkins/conformance/jenkins-conformance.properties",
-      "--script=test/e2e_node/jenkins/conformance/conformance-jenkins.sh"
+      "--deployment=node",
+      "--gcp-project=k8s-jkns-ci-node-e2e",
+      "--gcp-zone=us-central1-f",
+      "--node-args=--image-config-file=../test-infra/jobs/benchmark-config.yaml",
```
Comment (MHBauer) [inline jobs/config.json:9263]: couple years later I'm here wondering why the benchmark-config is used for the conformance suite?

**Pull request merged**